share/models/creative.py

```diff
@@ -43,6 +50,32 @@ class Meta(ShareObject.Meta):
         db_table = 'share_creativework'
         verbose_name_plural = 'Creative Works'
 
+    def defrankenize(self, *_, im_really_sure_about_this=False):
+        if not im_really_sure_about_this:
+            raise ValueError('You have to be really sure about this')
+
+        logger.info('Defrankenizing %r', self)
+
+        with transaction.atomic():
+            logger.info('Removing relations')
+            for field in AbstractCreativeWork._meta.get_fields():
+                if not field.one_to_many or field.name in ('changes', 'versions'):
+                    continue
+
+                logger.warning('Removing all %s', field.related_name)
+                relation = getattr(self, field.get_accessor_name())
+                num_deleted, stats = Change.objects.filter(id__in=relation.values_list('change_id', flat=True)).delete()
+                logger.warning('Deleted %d changes to remove %s', num_deleted, field.related_name)
+
+                assert num_deleted == stats.pop('share.Change', 0)
+
+                if stats:
+                    logger.error('Unexpectedly removed other rows, %r', stats)
+                    raise ValueError('Unexpectedly removed other rows, {!r}'.format(stats))
+
+            logger.info('Relations removed')
+            self.administrative_change(is_deleted=True, title='Defrankenized work')
```

Comment:
It would be, but I'd rather not deal with the migration and I like to think SSO will be merged soon enough that this won't be a problem.